conf/battle/items.conf

```diff
@@ -89,12 +89,9 @@ allow_consume_restricted_item: no
 // If the equip is compounded with restricted card(s), it ignores this check but still gives no effect.
 allow_equip_restricted_item: yes
 
-// Allow changing of equipment while interacting with NPCs? (Note 3)
-// 0: Disabled. (Default on renewal)
-// 1: Enabled on pre-renewal. (Default on pre-renewal)
-// 2: Enabled on renewal.
-// 3: 1+2
-item_enabled_npc: 1
+// Allow changing of equipment while interacting with NPCs? (Note 1)
+// Default on official servers: yes for Pre-renewal, no for Renewal
+//item_enabled_npc: yes
```

Comment:
why? 

//item_enabled_npc: yes

shouldn't be, 

item_enabled_npc: yes

without // ?

Reply:
the conf is using the default value when it doesn't exist in conf folder